src/engine.ts

```diff
@@ -55,20 +56,45 @@ export default class LuaEngine {
         }
     }
 
+    /**
+     * Executes Lua code from a string asynchronously.
+     * @param script - Lua script to execute.
+     * @returns A Promise that resolves to the result returned by the Lua script execution.
+     * @throws {Error}
+     */
     public doString(script: string): Promise<any> {
         return this.callByteCode((thread) => thread.loadString(script))
     }
 
+    /**
+     * Executes Lua code from a file asynchronously.
+     * @async
```

Comment:
No need to add @async with TSDoc

Reply:
Forgot to remove async

Reply:
please just remove this async and we are good to merge :)